src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(...args) {
+    if (args.length === 0) {
```

Comment:
compare with undefined
```suggestion
    if (args.length === undefined) {
```